### PR TITLE
Added the information about the list of URLs to be open for OpenSCAP …

### PR DESCRIPTION
**Pull request by suyogsainkar**
…scanning in the procedure for Adding an OpenShift Container Platform Provider. Also added the file from common to the Integration with OpenShift Container Platform guide.

BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1490879

### CODE REVIEW

**Comment by suyogsainkar** [comment]
@moolitayer @cben 
Hi Mooli/Beni, please would you or someone on your team mind taking a look at the steps and note under advanced settings in the section on Adding an OpenShift Container Platform Provider and let me know if it all makes sense? While this PR only addresses the info about the list of URLs for OpenSCAP scanning, some of the other recent updates there addresses the following bugs: 
https://bugzilla.redhat.com/show_bug.cgi?id=1490879
https://bugzilla.redhat.com/show_bug.cgi?id=1532450
https://bugzilla.redhat.com/show_bug.cgi?id=1532480 
Thanks! 
-Suyog

File: common/provider-ocp-add-container.adoc [hunk 1]
```diff
@@ -62,7 +62,12 @@ In OpenShift, the default deployment of the router generates certificates during
 +
 [NOTE]
 ====
-These settings download the image inspector container image from the registry and obtain the Common Vulnerabilities and Exposures (CVE) information (for effective scanning) via a proxy.
+These settings download the image inspector container image from the registry and obtain the Common Vulnerabilities and Exposures (CVE) information (for effective scanning) via a proxy. This requires {product-title_short} appliance to have external access to reach the Internet; the URLs required by {product-title_short} to be open for OpenSCAP scanning are:
```
Comment (cben) [inline common/provider-ocp-add-container.adoc]: "these settings download ... and obtain" is grammatically weird.
"these settings control downloading ... and obtaining" perhaps?

Comment (cben) [inline common/provider-ocp-add-container.adoc]: This doesn't really require CF to have access to *The* Internet; you could point them to local mirrors.
In fact being able to run behind a firewall without internet access was the main use case for adding them as settings.

Comment (suyogsainkar) [reply common/provider-ocp-add-container.adoc]: Fixed, thanks.

Comment (suyogsainkar) [reply common/provider-ocp-add-container.adoc]: Fixed, thanks.

**Comment by suyogsainkar** [comment]
@dayleparker Hi Dayle, please would you mind peer reviewing and merging this PR? Let me know what you think. Thanks for reviewing! Cheers, Suyog

**Comment by dayleparker** [comment]
Hi Suyog,

The updates look good so far. The only thing I'm not quite clear on, is what does it mean by "URLs required to be open"? Should you have these URLs open in a separate browser window, or are these used in the Image-Inspector fields in steps 11a-11e, or something else entirely? (Or is this obvious to someone who understands OpenSCAP scanning better?)

Also, is this link correct? It leads to an error message for me: https://cve.mitre.org/cgi-bin/cvename.cgi?name=

I think the merge conflict is due to my updates in that other bug about changing all the "ose" to "ocp", so I will resolve that when I merge this :)

Cheers,
Dayle

**Comment by suyogsainkar** [comment]
Hi Dayle, 

Thanks for your review and comments. I think opening URLs and ports is a fairly standard concept, especially for admins and other operators. Organizations may have configured a firewall to prevent accessing certain resources and links. So by opening those URLs, it allows traffic to proceed to these URLs through proxy. 

With regards to the actual URL - https://cve.mitre.org/cgi-bin/cvename.cgi?name=, by "opening" the URL through a proxy as in the step, it will allow any URL of that type (starting with that URL). The URL currently resolves in an error because it does not specify a CVE-ID at the end (we don't know what the ID will be), but when suffixed with an ID it will resolve correctly.

All that said, I've modified the text slightly to specify entering the URLs in the proxy fields - in step 11.a. Hope the edit makes it a bit clearer. :) Let me know what you think.

Thanks,
Suyog

**Comment by dayleparker** [comment]
Hey Suyog,
Thanks for the update - it looks more clear now, and what you said above makes total sense.
I tried fixing the merge conflict myself but whatever I did doesn't seem to be appearing here -- would you mind resolving the conflict please, and then I can hit merge?

It's in this file:
	both modified:   doc-Integration_with_OpenShift_Container_Platform/topics/provider-ocp-add-container.adoc

Cheers,
Dayle

**Comment by suyogsainkar** [comment]
@dayleparker Thanks Dayle, and no worries. After a few unsuccessful tries to resolve the conflict myself I was finally able to identify the issue. So although the conflict in the content was resolved, it seems the file doc-Integration_with_OpenShift_Container_Platform/topics/provider-ocp-add-container.adoc which you recently renamed (from provider-ose-add-container.adoc) as part of another bug was not updated/resolved here.
Also, with that change, the file name then matched exactly with another file having the same name in /common, and which is in fact included in the body of this file for the common procedure.:) To avoid confusion (and resolve the conflict), I've renamed it to add-ocp-provider.adoc. The conflict is now resolved; merging the PR to master. Cheers, Suyog

**Comment by dayleparker** [comment]
Thanks Suyog! I spotted that double-naming too but thought it was intentional (and glad you changed it). :)

**Pull request merged**